.github/workflows/build.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           java-version: 11
 
       - name: Clean & Build Jar
-        run: ./gradlew clean test jar
```

Comment:
We should keep `test`.
So in the future, if there is change, we will easier to find the problem.

Reply:
BTW, in the gradle there is `check` task which should be wider than just the `test`. Build is including the  `check`

Reply:
Yup. You are right.